src/Bonnier/WP/Cxense/Assets/Scripts.php

```diff
@@ -105,19 +105,23 @@ public function head_tags()
     private function objects_to_array($items)
     {
         if (!is_wp_error($items) && !empty($items)) {
-
             // Only one? Just return it
-            if(count($items) === 1) {
-                return $items[0]->name;
+            if(is_array($items)){
+                if(count($items) === 1) {
```

Comment:
You can have these two if's on one line if you want:
if(is_array($items) && count($items) === 1) {